mlflow/utils/search_utils.py

```diff
@@ -1846,3 +1814,109 @@ def _get_comparison(cls, comparison):
         comp["comparator"] = stripped_comparison[1].value
         comp["value"] = cls._get_value(comp.get("type"), comp.get("key"), stripped_comparison[2])
         return comp
+
+
+class SearchLoggedModelsUtils(SearchUtils):
+    NUMERIC_ATTRIBUTES = {
+        "creation_timestamp",
+        "creation_time",
```

Comment:
Do we need to support both `creation_timestamp` and `creation_time`?

Reply:
I'm not sure, we supported timestamp & timestamp_ms for example, it's just a little bit shorter

Reply:
Let's remove `creation_time`.